src/utils.js

```diff
@@ -44,7 +44,13 @@ export function conditionallyUpdateScrollbar() {
   }
 }
 
-export function mapToCssModules(className, cssModule) {
+let globalCssModule = {};
```

Comment:
Don't default this to an empty object, leave it undefined so that when nothing is provided `mapToCssModules` will do nothing.

Reply:
Made the change